### ISSUE TITLE
spkl: Unknown label gemerated from earlybound

### ISSUE THREAD

**Issue by AdiVlacic** [scottdurow/SparkleXrm#286]
Using the earlybound generator in spkl gives me an issue with labels on optionsets and statuscodes. The generated labels are all with the text unknown. Is it possible to change the language code for the earlybound generator? My CRM only has one langauge that is not english.

![image](https://user-images.githubusercontent.com/39899902/48487146-9ff30680-e81d-11e8-9682-da5539d1b0b9.png)

**Comment by scottdurow** [comment]
Yes - this is definitely down to the language support - need to get that fixed!

**Comment by JannyM85** [comment]
Any updates?
